lxd/daemon.go

```diff
@@ -518,27 +518,31 @@ func (d *Daemon) init() error {
 	/* Log expiry */
 	d.tasks.Add(expireLogsTask(d.State()))
 
-	/* set the initial proxy function and external auth based on config values in the DB */
+	/* Setup the proxy handler, external authentication and MAAS */
```

Comment:
While you're changing this should probably be using `//` comments.

Reply:
Same as the other, probably best to do in one shot in a separate commit.